arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/Either.kt

```diff
@@ -904,7 +904,7 @@ public sealed class Either<out A, out B> {
 
   @Deprecated(
     NicheAPI + "Prefer when or fold instead",
-    ReplaceWith(" fold({ MN.empty() }, f)")
+    ReplaceWith(" fold({ empty }, f)")
```

Comment:
I am a bit confused by these replacements. This won't result in correct code, right?

Reply:
Yes, it won't work. I'm writing an entry in the arrow web migration guide just to explain this. It's the only way to get rid of the `Monoid.empty` deprecated method.